modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceWidget.macro

```diff
@@ -304,6 +400,17 @@ definition {
 		Click(locator1 = "Button#EXPAND");
 	}
 
+	@summary = "This macro can be used for navigating to the Organization Management Widget entry."
+	macro searchAndGoToWidgetEntry(entryName = null) {
```

Comment:
Be more specific in the macro name as it is focused on the Organization Management widget